packages/fiori-freestyle-writer/test/__snapshots__/worklist.test.ts.snap

```diff
@@ -10033,15 +10033,21 @@ server:
       afterMiddleware: compression
       configuration:
         ignoreCertError: false # If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted
-        backend:
-          - path: /catalog-admin-noauth
-            url: https://fesamples-tooling.cfapps.sap.hana.ondemand.com
         ui5:
           path:
             - /resources
             - /test-resources
           url: https://ui5.sap.com
-          version: '' # The UI5 version, for instance, 1.78.1. Empty string means latest version
+          version: 1.78.16 # The UI5 version, for instance, 1.78.1. Empty string means latest version
```

Comment:
Should version be set here and not previously?

Reply:
This looks like a bug fix to me of a bug that we might have missed? Check https://github.com/SAP/open-ux-tools/blob/main/packages/fiori-freestyle-writer/test/basic.test.ts#L25 ... this should result with a fixed UI5 version in the generated `ui5.yaml`.

Reply:
Nice one. Yeah. looks good. Thanks